internal/sbi/processor/influence_data.go

```diff
@@ -48,7 +48,7 @@ func (p *Processor) ApplicationDataInfluenceDataGetProcedure(c *gin.Context, col
 
 func (p *Processor) ParseSnssaisFromQueryParam(snssaiStr string) []models.Snssai {
 	var snssais []models.Snssai
-	err := json.Unmarshal([]byte("["+snssaiStr+"]"), &snssais)
+	err := json.Unmarshal([]byte(snssaiStr), &snssais)
```

Comment:
@ian60509 Why does this line need to be changed?

Reply:
According to TS 29.571 Snssai definition : 
* pattern: `'^[A-Fa-f0-9]{6}$'`
I revised it to comply with the spec